test.js

```diff
@@ -4,5 +4,7 @@ var MAX_SAFE_INTEGER = require('./');
 
 test(function (t) {
 	t.assert(typeof MAX_SAFE_INTEGER === 'number');
+	t.is(MAX_SAFE_INTEGER + 1, MAX_SAFE_INTEGER + 2);
```

Comment:
haha... this is what happens when I make one-shot PR's by editing the file directly on Github ;) I'll update the PR